src/main/scala/it/pagopa/interop/purposeprocess/service/RiskAnalysisService.scala

```diff
@@ -1,16 +1,26 @@
 package it.pagopa.interop.purposeprocess.service
 
 import it.pagopa.interop.purposeprocess.model.riskAnalysisTemplate.RiskAnalysisFormConfig
+import it.pagopa.interop.tenantmanagement.client.model.TenantKind
+import it.pagopa.interop.tenantmanagement.client.model.TenantKind.{GSP, PA, PRIVATE}
 
 import scala.io.Source
 import spray.json._
 
 object RiskAnalysisService {
 
+  val riskAnalysisTemplatePath: String = "riskAnalysisTemplate/forms"
+
   // The Map key must correspond to the version field of the risk analysis form
-  val riskAnalysisForms: Map[String, RiskAnalysisFormConfig] = Map(
-    "1.0" -> loadRiskAnalysisFormConfig("riskAnalysisTemplate/forms/1.0.json"),
-    "2.0" -> loadRiskAnalysisFormConfig("riskAnalysisTemplate/forms/2.0.json")
+  val riskAnalysisForms: Map[TenantKind, Map[String, RiskAnalysisFormConfig]] = Map(
+    PA      -> Map(
+      "1.0" -> loadRiskAnalysisFormConfig(s"$riskAnalysisTemplatePath/${TenantKind.PA.toString}/1.0.json"),
+      "2.0" -> loadRiskAnalysisFormConfig(s"$riskAnalysisTemplatePath/${TenantKind.PA.toString}/2.0.json")
+    ),
+    PRIVATE -> Map(
+      "1.0" -> loadRiskAnalysisFormConfig(s"$riskAnalysisTemplatePath/${TenantKind.PRIVATE.toString}/1.0.json")
+    ),
+    GSP -> Map("1.0" -> loadRiskAnalysisFormConfig(s"$riskAnalysisTemplatePath/${TenantKind.GSP.toString}/1.0.json"))
```

Comment:
GSPs use the private risk analysis
```suggestion
    GSP -> Map("1.0" -> loadRiskAnalysisFormConfig(s"$riskAnalysisTemplatePath/${TenantKind.PRIVATE.toString}/1.0.json"))
```